docs/content/operations/api-reference.md

```diff
@@ -384,8 +390,44 @@ only want the active leader to be considered in-service at the load balancer.
 
 #### Tasks<a name="overlord-tasks"></a> 
 
+Note that all _interval_ URL parameters are ISO 8601 strings delimited by a `_` instead of a `/` 
+(e.g., 2016-06-27_2016-06-28).
+
 ##### GET
 
+* `/druid/indexer/v1/tasks`
+
+Retrieve list of tasks. Accepts query string parameters `state`, `datasource`, `createdTimeInterval`, `max`, and `type`.
+
+|Query Parameter |Description |
+|---|---|
+|`state`|filter list of tasks by task state, valid options are `running`, `complete`, `waiting`, and `pending`.|
+| `datasource`| return tasks filtered by Druid datasource.|
+| `createdTimeInterval`| return tasks created within the specified interval. | 
+| `max`| maximum number of `"complete"` tasks to return. Only applies when `state` is set to `"complete"`.|
+| `type`| filter tasks by task type.|
```

Comment:
I think there are too many, and it can't really be done exhaustively since extensions can add to this as well. Instead I think I'm going to link to [ingestion tasks doc](http://druid.io/docs/latest/ingestion/tasks.html) to at least point users in the right direction.